aea/agent.py

```diff
@@ -162,24 +157,15 @@ def timeout(self) -> float:
         return self._timeout
 
     @property
-    def loop_mode(self) -> str:
+    def loop_mode(self) -> Optional[str]:
```

Comment:
Why is it Optional; we should have a default loop mode...

Reply:
done